README.md

```diff
@@ -32,11 +32,9 @@ Assumes you have already setup and installed [Docker](https://docs.docker.com/en
 1. `git clone https://github.com/Islandora-CLAW/claw-docker.git`
 2. `cd claw-docker`
 3. `cp example.docker-compose.yml docker-compose.yml` (edit `docker-compose.yml`)
-4. `git submodule update --init --recursive`
-5. `cd commands`
-6. `./generate-env-files`
-7. `sudo ./build`
-8. `sudo docker-compose up`
+4. `cd commands`
+5. `./generate-env-files`
+6. `sudo docker-compose up`
```

Comment:
np, but be careful with that it will remove all existing volumes (that aren't currently attached to a running container). So all the data pretty much, which is Ok for dev environment, but would really ruin someones day in a non dev environment.